docs/en/serverless/quickstarts/collect-data-with-native-otlp.asciidoc

```diff
@@ -0,0 +1,159 @@
+* https://elastic.github.io/opentelemetry/edot-sdks/index.html[Elastic Distributions of OpenTelemetry (EDOT) SDKs]
+
+// I'm not entirely sure how to be more helpful here
+// Should we expand on some of these in this doc?
+
+That's it. You're now ready to leverage OTel in Elastic Observability.
+
+[discrete]
+==== (Optional) Install EDOT Collector
+
+Don't have a collector or SDK running? No problem. Spin up an EDOT collector in just a few steps:
+
+* https://elastic.github.io/opentelemetry/quickstart/serverless/k8s.html[Kubernetes Quickstart]
+* https://elastic.github.io/opentelemetry/quickstart/serverless/hosts_vms.html[Hosts & VMs Quickstart]
+
+// Commenting out Docker until the docs are ready
+// * https://elastic.github.io/opentelemetry/quickstart/serverless/docker.html[Docker Quickstart]
+
+[discrete]
+== Get creative with Discover
+
+**Discover allows you to quickly search and filter your data, get information about the structure of the fields in your data, and display your findings in a visualization.
+Find **Discover** in your {obs-serverless} project's UI under *Analyze / Discover*.
+
+Attributes and resource attributes are prefixed with `attributes.*` and `resource.attributes.*`.
+You can correlate all signals with a single `resource.attributes.*`.
+
+[role="screenshot"]
+image::images/resource-attrs.png[resource attributes]
+
+See {kibana-ref}/discover.html[Discover] to learn more.
+
+[discrete]
+== Monitor application performance
+
+The Applications UI allows you to monitor your software services and applications in real-time. You can visualize detailed performance information on your services, identify and analyze errors, and monitor host-level metrics.
+
+[discrete]
+== Check the health of your infrastructure
+
+To access the **Hosts** page, in your {obs-serverless} project, go to
+**Infrastructure** → **Hosts**.
+
+On the Hosts page, you can view health and performance metrics to help you quickly:
+
+* Analyze and compare hosts without having to build new dashboards.
+* Identify which hosts trigger the most alerts.
+* Troubleshoot and resolve issues quickly.
+
+[role="screenshot"]
+image::images/hosts-ui-otlp.png[resource attributes]
+
+See {observability-guide}/observability-analyze-hosts.html[Analyze and compare hosts] to learn more.
+
+[discrete]
+=== (Optional) Install the OpenTelemetry Assets integration
+
+Install the OpenTelemetry Assets integrations to access the "[OTEL][Metrics Kubernetes] Cluster Overview" dashboard.
+This dashboard 
+
+In your {obs-serverless} project, go to **Integrations** and toggle **Display beta integrations**.
+Search for "OpenTelemetry" and select and install **Kubernetes OpenTelemetry Assets**.
+
+// Does anyone have a cluster with relevant data that we can use to take a screenshot of this dashboard?
+
+[discrete]
+== Troubleshoot
+
+// are there specific troubleshooting scenarios we should call out?
```

Comment:
> are there specific troubleshooting scenarios we should call out?

Reply:
"Authorization: ApiKey <api-key>"

Reply:
@vigneshshanmugam any thoughts on this topic?

Reply:
Nothing specific comes to mind as you already called out the ApiKey format. I am also thinking if we can move the ratelimit scenario in troubleshooting as opposed to limitation. 

```
Error: too many requests

The Managed endpoint has per-project rate limits in place. If you hit this limit, reach out to our https://support.elastic.co[support team].
```

Maybe point the user to limitation doc if they are seeing any other errors? We can add other guides as followups as we learn more.